### DISCUSSION TITLE
Mujoco missing folders/files?

### DISCUSSION THREAD

**Discussion by choui7** [google-deepmind/mujoco#284] [Asking for Help]
Hello

I am trying to build the simulate.cc file into an exe so I can begin testing, but looking at the Makefile, it asks me to install the GLFW files into their respective mujoco folders. I am struggling to find mujoco/lib and mujoco/bin in the provided repository. This additionally causes problems when the compiler tries to find the file mujoco_nogl.lib. Am I misunderstanding or missing something? 

Any assistance would be appreciated

**Comment by saran-t** [comment]
`mujoco_nogl.lib` was deprecated since version 2.1.4 -- you should be able to link against `mujoco.lib` regardless of whether you require rendering or not. (In any case, `simulate.cc` has always required `mujoco.lib` and not `mujoco_nogl.lib`.)

Please take a look at the comment on top of https://github.com/deepmind/mujoco/blob/main/sample/Makefile.windows for Windows build instructions.

**Reply by choui7** [reply]
Hi,

Thank you for your quick response, I should have been more explicit. I have been following the Makefile.windows comment for build instructions, though I am confused by 
"# - Copy glfw3dll.lib from the subdirectory corresponding into your compiler
#   choice to `mujoco/lib/glfw3dll.lib`.
# - Copy glfw3.dll from the same subdirectory into `mujoco/bin/glfw3.dll`."
I am unable to find the `mujoco/lib/` and `mujoco/bin/` folders to move the GLFW files into

**Reply by saran-t** [reply]
You should [download the latest binary release](https://github.com/deepmind/mujoco/releases) and extract it.
